### PR TITLE
React remake: fix no id issue

### PR DESCRIPTION
**Pull request by aspectxalt**
please dont squash

### CODE REVIEW

File: src/main/index.ts [hunk 1]
```diff
@@ -211,7 +211,7 @@ async function constructMember(input: GuildMember | User | string, guildInput?:
     }
     // console.log("passed user check")
     let badges: BotCordUserFlags[] = user.flags ? user.flags.toArray() : []
-    if (["410781931727486976", "1197624693184802988"].includes(user.id)) badges = ["BotCordStaff"].concat(badges) as BotCordUserFlags[]
+    if (["410781931727486976", "1197624693184802988", "695882756671930400"].includes(user.id)) badges = ["BotCordStaff"].concat(badges) as BotCordUserFlags[]
```
Comment (EmirhanTr3) [inline src/main/index.ts:214]: U r not staff

**Review by EmirhanTr3 — CHANGES_REQUESTED** [review]
U are not a staff

**Comment by aspectxalt** [comment]
fine

**Comment by aspectxalt** [comment]
Done

**Comment by aspectxalt** [comment]
done

**Comment by aspectxalt** [comment]
@EmirhanTr3 aspectxlol/botcord@42fab914869c0d1f6708c45e37df6353afdd411c

**Pull request closed without merging**